luminaire/model/window_density.py

```diff
@@ -247,7 +247,7 @@ def _call_training(self, df=None, window_length=None, imputed_metric=None, detre
         agg_data_model = dict()
         agg_data = dict()
 
-        past_model = kwargs['past_model']
+        past_model = kwargs['past_model'] if 'past_model' in kwargs else None
```

Comment:
`past_model = kwargs.get('past_model')` # return value of kwarg key or None if key is not present